### ISSUE TITLE
NWjs User Folder Bug?

### ISSUE THREAD

**Issue by corlenbelspar** [Scirra/Construct-3-bugs#3735]
## Problem description

I seem to recall NWjs.UserFolder would return the full path including the drive letter, but it doesn't now.

## Attach a .c3p

[Userfolderbug.zip](https://github.com/Scirra/Construct-3-bugs/files/4410960/Userfolderbug.zip)

## Steps to reproduce

1. Just run it and read the text object I created.

## Observed result

The drive letter is missing from the path. This results in it not working with file read/write actions.

## Expected result

I thought it used to return the drive letter too in the path.

## More details

**Affected browsers/platforms:** NWjs

**First affected release:** r193

## System details

<details><summary>View details</summary>

Platform information
Browser: NW.js
Browser version: 0.44.0
Browser engine: Chromium
Browser architecture: 64-bit
Context: nwjs
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 16
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (@ac9418ba9c3bd7f6baaffa0b055dfe147e0f8364) NWjs/0.44.0 (like Chrome/80.0.3987.87)
C3 release: r193 (beta)
Language setting: en-US

Local storage
Storage quota (approx): 92 gb
Storage usage (approx): 117 mb (0.1%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Google Inc.
Renderer: ANGLE (NVIDIA GeForce GTX 1050 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:

EXT_color_buffer_float
EXT_disjoint_timer_query_webgl2
EXT_float_blend
EXT_texture_filter_anisotropic
KHR_parallel_shader_compile
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
OVR_multiview2
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by AshleyScirra** [comment]
Actually, it never included the drive letter. Construct just returns the value from the underlying node.js implementation, and for some reason it doesn't include the drive letter. This doesn't seem to matter, though: features involving such paths still seem to work fine - for example you can list all the files in that folder, without a drive letter, and still get a listing of all the files in the user folder. It's difficult to add in the right drive letter, because if we get it from somewhere else, we don't actually have a guarantee that the drive letter is the same one as is used for the user folder.

Basically this is a node.js issue, but it doesn't seem to matter, since Windows figures out the path anyway.
